DCS-SR-Client/UI/RadioOverlayWindow/RadioOverlayTenTransparent.xaml.cs

```diff
@@ -0,0 +1,571 @@
+            Radio7.RadioActive.Opacity = e.NewValue;
+            Radio7.RadioVolume.Opacity = e.NewValue;
+            //Radio 8
+            Radio7.RadioEnabled.Opacity = e.NewValue;
+            Radio7.RadioTextGroup.Opacity = e.NewValue;
+            Radio7.RadioLabel.Opacity = e.NewValue;
+            Radio7.RadioFrequency.Opacity = e.NewValue;
+            Radio7.RadioMetaData.Opacity = e.NewValue;
+            Radio7.TransmitterName.Opacity = e.NewValue;
+            Radio7.RadioActive.Opacity = e.NewValue;
+            Radio7.RadioVolume.Opacity = e.NewValue;
+            //Radio 8
+            Radio8.RadioEnabled.Opacity = e.NewValue;
+            Radio8.RadioTextGroup.Opacity = e.NewValue;
+            Radio8.RadioLabel.Opacity = e.NewValue;
+            Radio8.RadioFrequency.Opacity = e.NewValue;
+            Radio8.RadioMetaData.Opacity = e.NewValue;
+            Radio8.TransmitterName.Opacity = e.NewValue;
+            Radio8.RadioActive.Opacity = e.NewValue;
+            Radio8.RadioVolume.Opacity = e.NewValue;
+            //Radio 9
+            Radio9.RadioEnabled.Opacity = e.NewValue;
+            Radio9.RadioTextGroup.Opacity = e.NewValue;
+            Radio9.RadioLabel.Opacity = e.NewValue;
+            Radio9.RadioFrequency.Opacity = e.NewValue;
+            Radio9.RadioMetaData.Opacity = e.NewValue;
+            Radio9.TransmitterName.Opacity = e.NewValue;
+            Radio9.RadioActive.Opacity = e.NewValue;
+            Radio9.RadioVolume.Opacity = e.NewValue;
+            //Radio 10
+            Radio10.RadioEnabled.Opacity = e.NewValue;
+            Radio10.RadioTextGroup.Opacity = e.NewValue;
+            Radio10.RadioLabel.Opacity = e.NewValue;
+            Radio10.RadioFrequency.Opacity = e.NewValue;
+            Radio10.RadioMetaData.Opacity = e.NewValue;
+            Radio10.TransmitterName.Opacity = e.NewValue;
+            Radio10.RadioActive.Opacity = e.NewValue;
+            Radio10.RadioVolume.Opacity = e.NewValue;
+            //Vox and Intercom
+            Intercom.Opacity = e.NewValue;
+            //Footer
+            buttonShow.Opacity = e.NewValue;
+            textBackground.Opacity = e.NewValue;
+            textText.Opacity = e.NewValue;
+        }
+
+        private void backgroundOpacitySlider_ValueChanged(object sender, RoutedPropertyChangedEventArgs<double> e)
+        {
+            Background.Opacity = e.NewValue;
+            
+            //Opacity = e.NewValue;
+        }
+
+        private void containerPanel_SizeChanged(object sender, SizeChangedEventArgs e)
+        {
+            //force aspect ratio
+            CalculateScale();
+
+            WindowState = WindowState.Normal;
+            Logger.Debug("Size Changed Transparent Panel");
+        }
+
+
+        private void CalculateScale()
+        {
+            var yScale = ActualHeight / RadioOverlayWin.MinHeight;
+            var xScale = ActualWidth / RadioOverlayWin.MinWidth;
```

Comment:
This line in every Panel is `ActualWidth / RadioOverlayWin.MinHeight;` maybe change it to this. To remove the growing size.
Its just an idea I had, because I changed this and never changed it back.